lib/srv/authhandlers.go

```diff
@@ -65,6 +80,30 @@ type AuthHandlers struct {
 	Clock clockwork.Clock
 }
 
+// AuthHandlers are common authorization and authentication related handlers
+// used by the regular and forwarding server.
+type AuthHandlers struct {
+	*log.Entry
```

Comment:
Nit: Could you also unembed the logger please while you're here if it's not too much refactoring?

```go
type AuthHandlers struct {
  log *log.Entry
  ...
}
```